packages/flutter/lib/src/cupertino/form_section.dart

```diff
@@ -205,10 +199,8 @@ class CupertinoFormSection extends StatelessWidget {
               fontSize: 13.0,
               color: CupertinoColors.secondaryLabel.resolveFrom(context),
             ),
-            child: Padding(
-              padding: _kFormDefaultHeaderMargin,
-              child: header,
-            ));
+            child: header!,
```

Comment:
This would very likely cause crashes, because it's possible that a header isn't provided and we don't enforce that it is. It would be better to do something like `child: header ?? Container()', or give header some kind of empty default. I do not think this file should be changed though.

Reply:
We have null check before this line, so it should always have a value here
https://github.com/flutter/flutter/pull/129065/files/cb398cb59ae47f552be75af9c1dcb6c1d3f79ae5#diff-23b84c50ff8076139e5ba22d7a3e264ab29b06cd0efb0360dd9673ab257baefcR195

Reply:
My apologies, you're right.